clients/admin-ui/cypress/fixtures/dictionary-entries.json

```diff
@@ -176,7 +176,8 @@
       "domains": null,
       "logo": null,
       "gvl_version": "3",
-      "cookies": []
+      "cookies": [],
+      "vendor_deleted_date": "2022-06-08T00:00:00+00:00"
```

Comment:
not true (i.e. this vendor doesn't actually have a `vendor_deleted_date` populated), but figured it was good to improve our test fixtures coverage of different scenarios